src/main/java/gwt/jsonix/marshallers/xjc/plugin/utils/ClassNameUtils.java

```diff
@@ -22,8 +22,6 @@
 
     private static final String SEPARATOR = "__";
 
-    private static final String CLASS_PREFIX = "JSI";
```

Comment:
@jomarko 
Hi. Since you are there, could you please remove also the commented parameter at line 33. Thx

Reply:
@gitgabrio sure, done, thank you